src/js/components/search/FilterExpandButton.jsx

```diff
@@ -25,10 +25,32 @@ export default class FilterExpandButton extends React.Component {
             icon = <Icons.AngleDown />;
         }
 
+        let clickEvent = '';
+        switch (this.props.name) {
+            case 'Search':
+                clickEvent = null;
+                break;
+            case 'Budget Categories':
+                clickEvent = null;
+                break;
+            case 'Award ID':
+                clickEvent = null;
+                break;
+            case 'Award Amount':
+                clickEvent = null;
+                break;
+            case 'Other Award Items':
+                clickEvent = null;
+                break;
+            default:
+                clickEvent = this.props.toggleFilter;
+                break;
```

Comment:
1) See comment in RankVisualizationScopeButton about the `disabled` attribute to disable a button
2) The FilterExpandButton component should not care what filter it belongs to. The parent component should pass a prop to it indicating its enable/disabled state, which in turn sets the `<button>` element's `disabled` attribute